package.json

```diff
@@ -133,12 +133,16 @@
 		"showdown": "^1.9.0"
 	},
 	"devDependencies": {
+		"dictionary-en-gb": "^2.1.1",
```

Comment:
W3C work is in US English, not British English. We should do the same so we don't have to adjust grammar of rules when we propose them to the W3C.